locations/jclouds/src/test/java/brooklyn/location/jclouds/BrooklynMachinePoolLiveTest.java

```diff
@@ -98,6 +98,6 @@ public void buildClaimAndDestroy() {
     
 
     private JcloudsLocation resolve(String spec) {
-        return (JcloudsLocation) managementContext.getLocationRegistry().resolve(JcloudsLocationResolver.JCLOUDS+":"+spec);
+        return (JcloudsLocation) managementContext.getLocationRegistry().resolve("jclouds:"+spec);
```

Comment:
There are so many classes that hardcode "jclouds:..." that I think it's a minor one, but I'll switch it for `new JcloudsLocationResolver().getPrefix()`